### PR TITLE
Updating news and authors files

### PR DESCRIPTION
**Pull request by shamisp**
- Adding v1.4.0 update
- Porting v1.3.1 news from 1.3.x branch
- Updated contributors list

### CODE REVIEW

**Comment by shamisp** [comment]
@yosefe let's add whatever missing

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7614/ for details (Mellanox internal link).

**Comment by hoopoepg** [comment]
failed due to memchecker zombie

**Comment by hoopoepg** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7615/ for details (Mellanox internal link).

**Comment by shamisp** [comment]
@hoopoepg what is wrong with regression ?

**Comment by hoopoepg** [comment]
@shamisp bgate is out of service, and all engineers who can fix it are on new year holidays :)

**Comment by hoopoepg** [comment]
bot:mlx:retest

**Comment by hoopoepg** [comment]
ppc is not stable too :(

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7619/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
looks like there is a memory leak in ucp requests mpool which causes the failures, checking it

**Comment by yosefe** [comment]
bot:retest

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7622/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7627/ for details (Mellanox internal link).

File: NEWS [hunk 1]
```diff
@@ -7,7 +7,46 @@
...
-## 1.3.0 (February 15, 2018)
+## Current
+
+## 1.4.0-rc1(September 7, 2018)
+
+Features:
+- Improved support for installation with latest ROCm
+- Improved support for latest rdma-core
+- Adding support for CUDA IPC for inter node GPU
+- Added support for latest Mellanox devices
```
Comment (yosefe) [inline NEWS]: - Support for multiple connections between same pair of endpoints
- Support large worker address for client/server connection establishment and INARRD_ANY
- Bitwise atomics support
@bureddy anything more for cuda?

File: NEWS [hunk 2]
```diff
@@ -7,7 +7,46 @@
...
+
+Bugfixes:
+- Performance fixes for rendezvous' protocol
```
Comment (yosefe) [inline NEWS]: which one is it?

Comment (shamisp) [reply NEWS]: I think it was about alignment

File: NEWS (hunk 1 above)
Comment (evgeny-leksikov) [reply NEWS]: @yosefe same pair of workers, not endpoint

Comment (yosefe) [reply NEWS]: right

File: NEWS (hunk 2 above)
Comment (yosefe) [reply NEWS]: not sure what you mean here, let's remove

Comment (shamisp) [reply NEWS]: https://github.com/openucx/ucx/pull/2830

File: NEWS [hunk 3]
```diff
@@ -7,7 +7,46 @@
...
 
-## 1.3.0 (February 15, 2018)
+## Current
+
+## 1.4.0-rc1(September 7, 2018)
+
+Features:
+- Improved support for installation with latest ROCm
+- Improved support for latest rdma-core
+- Adding support for CUDA IPC for inter node GPU
```
Comment (bureddy) [inline NEWS]: should be intra-node.

File: NEWS (hunk 1 above)
Comment (bureddy) [reply NEWS]: see if this can be added
-  cuda memory allocation cache for mem type detection

**Comment by yosefe** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7650/ for details (Mellanox internal link).

File: NEWS [hunk 4]
```diff
@@ -1,13 +1,64 @@
...
+
+## 1.4.0-rc1(September 7, 2018)
+
+Features:
+- Improved support for installation with latest ROCm
+- Improved support for latest rdma-core
+- Adding support for CUDA IPC for intra-node GPU
+- Added support for CUDA memory allocation cache for mem-type detection
+- Added support for latest Mellanox devices
+- Added support for multiple connections between same pair of endpoints
```
Comment (evgeny-leksikov) [inline NEWS]: same pair of workers

File: NEWS [hunk 5]
```diff
@@ -1,13 +1,64 @@
...
 
-## 1.3.0 (February 15, 2018)
+## Current
+Feaures:
+- New emulation mode enabling full UCX functionality (Atomic, Put, Get)
+  over TCP and RDMA-CORE interconnects that to do implement full RDMA semantics.
+- Non-blocking API for all one-sided operations. All blocking communication APIs marked
+  as depricated.
+- New client/server API
+- Added CUDA support for stream API
```
[the remark this answers is not in the document]
Comment (shamisp) [reply NEWS]: This is under current section that represents masers...

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7724/ for details (Mellanox internal link).

**Comment by shamisp** [comment]
@yosefe I would like to publish RC1. Are you okay to merge it into the master, backport to v1.4.x and get the RC out of the door ?

File: NEWS [hunk 6]
```diff
@@ -1,13 +1,64 @@
...
 ##
 ## See file LICENSE for terms.
 ##
 #
 
-## 1.3.0 (February 15, 2018)
+## Current
+Feaures:
+- New emulation mode enabling full UCX functionality (Atomic, Put, Get)
+  over TCP and RDMA-CORE interconnects that to do implement full RDMA semantics.
```
Comment (yosefe) [inline NEWS]: "to do" -> "do not"

File: NEWS [hunk 7]
```diff
@@ -1,13 +1,64 @@
...
+- Non-blocking API for all one-sided operations. All blocking communication APIs marked
+  as depricated.
+- New client/server API
```
Comment (yosefe) [inline NEWS]: New client/server connection establishment API

File: NEWS [hunk 8]
```diff
@@ -1,13 +1,64 @@
...
+
+Features:
+- Improved support for installation with latest ROCm
+- Improved support for latest rdma-core
+- Adding support for CUDA IPC for intra-node GPU
+- Added support for CUDA memory allocation cache for mem-type detection
+- Added support for latest Mellanox devices
+- Added support for multiple connections between the same pair of workers
+- Added support large worker address for client/server connection establishment
+  and INARRD_ANY
```
Comment (yosefe) [inline NEWS]: INADDR_ANY

File: NEWS [hunk 9]
```diff
@@ -1,13 +1,64 @@
...
+- Added support for bitwise atomics operations
+
+Bugfixes:
+- Performance fixes for rendezvous' protocol
```
Comment (yosefe) [inline NEWS]: rendezvous - without 's

File: NEWS [hunk 10]
```diff
@@ -1,13 +1,64 @@
...
+- Clang support fixes
+- Self tl multi-rail fix
+- Multiple minor bugfixes (full list on githib)
+
+Tested configurations:
+- InfiniBand: MLNX_OFED 4.2-4.4, distribution inbox drivers, rdma-core
+- CUDA: gdrcopy 1.2, cuda 9.1.85
+- XPMEM: 2.6.2
+- KNEM: 1.1.2
+- Multiple bugfixes (full list on githib)
```
Comment (yosefe) [inline NEWS]: githib -> github

File: NEWS [hunk 11]
```diff
@@ -1,13 +1,64 @@
...
+## Current
+Feaures:
+- New emulation mode enabling full UCX functionality (Atomic, Put, Get)
+  over TCP and RDMA-CORE interconnects that to do implement full RDMA semantics.
+- Non-blocking API for all one-sided operations. All blocking communication APIs marked
+  as depricated.
+- New client/server API
+- Added CUDA support for stream API
+
+## 1.4.0-rc1(September 7, 2018)
```
Comment (yosefe) [inline NEWS]: add space before (

File: NEWS [hunk 12]
```diff
@@ -1,13 +1,64 @@
...
+- Limit IB CQE size for specific ARM boards
+- RPM: explicitly set gcc-c++ as requirement
+- Multiple bugfixes (full list on githib)
+Tested configurations:
+- InfiniBand: MLNX_OFED 4.2, inbox OFED drivers.
+- CUDA: gdrcopy 1.2, cuda 9.1.85
+- XPMEM: 2.6.2
+- KNEM: 1.1.2
+
+## 1.3.0(February 15, 2018)
```
Comment (yosefe) [inline NEWS]: add space before (

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7759/ for details (Mellanox internal link).

File: NEWS (hunk 11 above)
Comment (yosefe) [reply NEWS]: @shamisp this is not fixed

**Comment by shamisp** [comment]
@yosefe - good to go ?

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7775/ for details (Mellanox internal link).

**Pull request merged**